mysql-test/r/bug96677.result

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE t1(id INT);
+INSERT INTO t1 VALUES (1);
+SELECT id AS foobar FROM t1 GROUP BY id LIMIT 1 INTO @id_var1 FOR UPDATE;
```

Comment:
On upstream, this first select fails with ER_SYNTAX_ERROR, so that's all the test does.